docker-compose.yaml

```diff
@@ -0,0 +1,181 @@
+# version is removed as per: https://github.com/docker/cli/pull/3445
+services:
+  # If you plan to deploy where a proxy is already in place, feel free to remove the traefik service
+  traefik:
+    image: docker.io/library/traefik:2.9
+    command:
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.websecure.address=:443"
+      - "--entrypoints.web.http.redirections.entrypoint.scheme=https"
+      - "--entrypoints.web.http.redirections.entrypoint.permanent=true"
+      - "--certificatesresolvers.letsencrypt.acme.httpchallenge.entrypoint=web"
+      - "--certificatesresolvers.letsencrypt.acme.email=${ACME_EMAIL}"
+      - "--certificatesresolvers.letsencrypt.acme.storage=/letsencrypt/acme.json"
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "letsencrypt:/letsencrypt"
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      # - "/run/podman/podman.sock:/var/run/docker.sock:ro"
+    restart: unless-stopped
+
+  crux:
+    container_name: dyrectorio_crux
+    image: ghcr.io/dyrector-io/dyrectorio/web/crux:latest
+    command:
+    - serve
+    environment:
+      - TZ=${TIMEZONE:-Europe/Budapest}
+      - DATABASE_URL=postgresql://crux:${CRUX_POSTGRES_PASSWORD}@crux-postgres:5432/crux?schema=public
+      - KRATOS_ADMIN_URL=http://kratos:4434
+      - KRATOS_URL=http://kratos:4433
+      - CRUX_UI_URL=https://${DOMAIN}
+      - CRUX_AGENT_ADDRESS=${DOMAIN}:443
+      - JWT_SECRET=${CRUX_SECRET}
+      - FROM_NAME="${DOMAIN} - dyrector.io"
+      - FROM_EMAIL=dyrectorio@${DOMAIN}
+      - SMTP_URI=${SMTP_URI}
+      # If you don't have a mailserver you can use the provided mailslurper
+      # but keep in mind, this one is insecure, only use it at your own risk!
+      # - SMTP_URI=smtps://test:test@mailslurper:1025/?skip_ssl_verify=true&legacy_ssl=true
+    depends_on:
+      crux-postgres:
+        condition: service_started
+      crux-migrate:
+        condition: service_completed_successfully
+      proxy:
```

Comment:
service "crux-ui" depends on undefined service proxy: invalid compose project

Reply:
It was renamed on the test-env after running into some issues with container naming. Thanks, fixed.